site/docs/v1/tech/tutorials/start-and-stop-pre-1-37.adoc

```diff
@@ -0,0 +1,132 @@
+---
+layout: doc
+title: Start and Stop FusionAuth
+description: Learn how to start and stop the FusionAuth services
+navcategory: getting-started
+---
+
+== Start and Stop FusionAuth
+
+:137_link: /docs/v1/tech/tutorials/start-and-stop
+[WARNING.warning]
+====
+include::docs/v1/tech/shared/_pre-1-37-warning.adoc[]
+====
+
+If you are using the FusionAuth server version, may need to manually start or stop FusionAuth periodically. If you find yourself in this situation you'll be glad you know how.
+
+[NOTE]
+====
+Unless you are hosting and managing FusionAuth these instructions do not apply to you. FusionAuth manages and monitors instances hosted in our private
+cloud and will restart FusionAuth if necessary.
+====
+
+
+[NOTE]
+====
+*Note* that you *MUST* start the FusionAuth Search before starting the FusionAuth App service. You should also shutdown the Search Engine last. FusionAuth
```

Comment:
This isn't that strong of a requirement any longer since we will wait. We do recommend you start search first.